.ci/containers/magic-modules-dev/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/graphite-docker-images/downstream-builder:latest
+WORKDIR /magic-modules
+ENTRYPOINT ["/usr/bin/make"]
```

Comment:
The container spends time fetching dependencies for tpgtools. 
This is duplicate work that can be avoided.

We'll likely have to change downstream builder to fetch these dependencies and update the trigger to work off the go.sum file.

Reply:
good call! I'll take a quick look, but worst case would a follow-up PR work?

Reply:
I think follow-up in a separate issue?